src/main/modules/nunjucks/index.ts

```diff
@@ -126,6 +126,8 @@ export class Nunjucks {
     nunjucksEnv.addGlobal('HowMuchPaidClaimantOption', HowMuchPaidClaimantOption)
     nunjucksEnv.addGlobal('Service', Service)
     nunjucksEnv.addGlobal('cookieText', `GOV.UK uses cookies make the site simpler. <a href="${AppPaths.cookiesPage.uri}">Find out more about cookies</a>`)
+    nunjucksEnv.addGlobal('moneyClaims', `Money Claims`)
```

Comment:
Please rename this to be `serviceName` or similar. A `moneyClaims` = `Money Claims` does not look good.